lib/src/model/hms_android_pip_controller.dart

```diff
@@ -3,6 +3,31 @@ import 'package:hmssdk_flutter/src/service/platform_service.dart';
 
 /// [HMSAndroidPIPController] is used to setup and start the PIP in android. To know more visit [here](https://www.100ms.live/docs/flutter/v2/advanced-features/pip-mode).
 class HMSAndroidPIPController {
+  ///setup PIP is used to setup PIP in Android devices.
+  ///**parameters**:
+  ///
+  /// **autoEnterPip** - Enable [autoEnterPip] will start pip mode automatically when app minimized. Default value is `true`
```

Comment:
is the default autoEnterPIP value true or false?
in the `HMSPipAction.kt` code above, it's mentioned as `false`